frontend/storage/common.ts

```diff
@@ -62,5 +68,12 @@ export const GameVisibilityActions = {
 		};
 	},
 };
-
-export type AppActions = SetBreadcrumbAction | SetGameVisibilityAction | SetEveryGameVisibilityAction;
+export const DownloadActions = {
+	setGames: function(games: Array<Game>): SetGamesAction {
```

Comment:
Ditto here, why not `Game[]`? I get the impression that's what we've been using for the most part